apps/frontend/src/libs/hooks/use-modal-click-outside/use-modal-click-outside.hook.ts

```diff
@@ -0,0 +1,25 @@
+import { type RefObject, useEffect } from "react";
```

Comment:
In our project, we import react hooks from `~/libs/hooks/hooks.js`
```suggestion
import { useEffect } from "~/libs/hooks/hooks.js";
```